build-tools/packages/build-tools/src/repoPolicyCheck/handlers/assertShortCode.ts

```diff
@@ -209,7 +244,7 @@ function writeShortCodeMappingFile() {
 
 // Auto-generated by policy-check in @fluidframework/build-tools.
 
-export const shortCodeMap = ${JSON.stringify(mapContents, null, 4)};
+export const shortCodeMap = ${JSON.stringify(mapContents, null, "\t")};
```

Comment:
nit: ideally whitespace would be configurable, but as you noted elsewhere, this is hardly the only thing that would better live in a config somewhere.